development/includes/_hero.pug

```diff
@@ -0,0 +1,5 @@
+div.hero
+  img
+  div
```

Comment:
remove this random div. If you have a style attached to a random div, you'll need to create a class for it.